test/cljs/rems/administration/test_create_catalogue_item.cljs

```diff
@@ -46,3 +46,105 @@
     (testing "missing form"
       (is (nil? (build-request (assoc form :form nil)
                                languages))))))
+
+(deftest resource-label-test
```

Comment:
I think this test has some problems that we could work on:
- First of all, this test is massive. We should not need to add almost 100 lines of example data to test this tiny feature. How about we reduce the resource test data to only what one call to `resource-label` actually needs? Second possibility is to refactor or extract the complex logic in another manner so that testing it becomes easier. Another possibility is writing this level of test as an integration test with real created resources. I think we should do at least the first one.
- The expected expressions contain a mix of language and unlocalized keys. I think it smells of wrong level of testing or functions. I think we can live with this for now.
- Why does the test of `resource-label` function need to use `map` and call the function to several resources? Would it not be better to e.g. test one resource at a time each in its own `is` expression with possible explanation as to what case it is?
- Why does it need to test all the different languages and why is not one enough?

Reply:
I agree with you that the test could use some clean-up. The pointer is pretty clear. I ll work on the test data a bit more and come back to you.

Reply:
"Why does it need to test all the different languages and why is not one enough? " - in the previous iterations I was not sure whether all license features were localized so adding that made sense

Reply:
"First of all, this test is massive. We should not need to add almost 100 lines of example data to test this tiny feature. How about we reduce the resource test data to only what one call to resource-label actually needs? " - done

Reply:
Why does the test of resource-label function need to use map and call the function to several resources? - isn't the entire purpose of the function to iterate through the selections?

Reply:
First of all you are testing `resource-label` i.e. the label for a *single resource*. Therefore the test should test that. And that does not iterate through selections, right?